middleware/packet-forward-middleware/testing/simapp/app.go

```diff
@@ -745,6 +753,34 @@ func (app *SimApp) SimulationManager() *module.SimulationManager {
 	return app.sm
 }
 
+func (app *SimApp) setupUpgradeHandlers() {
+	app.UpgradeKeeper.SetUpgradeHandler(
+		upgrades.V2,
+		upgrades.CreateV2UpgradeHandler(app.mm, app.configurator, app.ParamsKeeper, app.ConsensusParamsKeeper, app.PacketForwardKeeper),
+	)
+}
+
+// setupUpgradeStoreLoaders sets all necessary store loaders required by upgrades.
+func (app *SimApp) setupUpgradeStoreLoaders() {
```

Comment:
is this still necessary since the entire body is commented out anyways?

Reply:
per your other comment about renaming, yes this is required. I had left this for reference, but will be used now in the v2 rename migration